draft-ietf-quic-transport.md

```diff
@@ -3254,8 +3254,8 @@ The STREAM_ID_BLOCKED frame contains a single field.
 
 Stream ID:
 
-: A variable-length integer indicating the highest stream ID that the sender
-  was permitted to open.
+: A variable-length integer indicating the stream ID of the stream that the
+  sender was not able to open due to the maximum stream ID limit.
```

Comment:
To avoid ambiguity, would it make sense to clarify that the value is "the maximum stream ID among the streams that the sender is willing to open"?

Consider the case of a client that received from server a `initial_max_bidi_stream` of zero. If the client want to open two bidirectional streams (i.e. stream ID 0 and 4), should it send `STREAM_ID_BLOCKED` frame with stream ID of 0? My intent is to clarify that sending 4 in allowed such case.

Reply:
Done. PHAL.